specification/playwrighttesting/PlaywrightTesting.Management/main.tsp

```diff
@@ -0,0 +1,175 @@
+  @visibility("read")
+  @doc("The Playwright testing dashboard URI for the account resource.")
+  dashboardUri: url;
+
+  @doc("The regionalAffinity feature enablement status for the account.")
+  regionalAffinity?: EnablementStatus = EnablementStatus.Enabled;
+
+  @doc("The scalableExecution feature enablement status for the account.")
+  scalableExecution?: EnablementStatus = EnablementStatus.Enabled;
+
+  @doc("The reporting feature enablement status for the account.")
+  reporting?: EnablementStatus = EnablementStatus.Disabled;
+
+  @visibility("read")
+  @doc("The status of the last operation.")
+  provisioningState?: ProvisioningState;
+}
+
+@doc("The enablement status of a feature.")
+enum EnablementStatus {
+  @doc("The feature is Enabled.")
+  Enabled,
+
+  @doc("The feature is Disabled.")
+  Disabled,
+}
+
+@doc("The status of the current operation.")
+@Azure.Core.lroStatus
+enum ProvisioningState {
+  ...ResourceProvisioningState,
+
+  @doc("Deletion in progress")
+  Deleting,
+
+  @doc("Change accepted for processing")
+  Accepted,
+}
+
+@armResourceOperations
+interface Accounts
+  extends TrackedResourceOperations<Account, AccountProperties> {}
+
+#suppress "@azure-tools/typespec-azure-resource-manager/arm-resource-path-segment-invalid-chars" "Existing Template"
+#suppress "@azure-tools/typespec-azure-resource-manager/arm-resource-key-invalid-chars" "Existing template"
+@doc("The location of quota resource")
+@subscriptionResource
+model Location {
+  @doc("The location of quota in ARM Normalized format like eastus, southeastasia etc.")
+  @key("location")
+  @path
+  @segment("locations")
+  @visibility("query", "read", "update")
+  locationName: string;
+}
+
+#suppress "@azure-tools/typespec-azure-resource-manager/arm-resource-path-segment-invalid-chars" "Existing Template"
+#suppress "@azure-tools/typespec-azure-resource-manager/arm-resource-key-invalid-chars" "Existing template"
+@doc("A quota resource")
+@parentResource(Location)
+model Quota is ProxyResource<QuotaProperties> {
+  @doc("The quota bucket name.")
+  @key("name")
+  @path
+  @segment("quotas")
+  name: QuotaBucketNames;
+}
+
+@doc("The quota bucket name.")
+enum QuotaBucketNames {
+  @doc("The test execution quota.")
+  TestExecution,
+}
+
+@doc("Quota properties")
+model QuotaProperties {
+  @doc("The free-trial quota.")
+  freeTrial?: FreeTrialProperties;
+
+  @visibility("read")
+  @doc("The status of the last operation.")
+  provisioningState?: ProvisioningState;
+}
+
+@doc("The free-trial properties")
+model FreeTrialProperties {
+  @doc("The playwright account resource id.")
+  @visibility("read")
+  accountId: uuid;
+
+  @doc("The free-trial createdAt utcDateTime.")
+  @visibility("read")
+  createdAt: utcDateTime;
+
+  @doc("The free-trial expiryAt utcDateTime.")
+  @visibility("read")
+  expiryAt: utcDateTime;
+
+  @doc("The free-trial allocated limit value.")
+  @visibility("read")
+  allocatedValue: int32;
+
+  @doc("The free-trial used value.")
```

Comment:
>The free-trial used value [](http://example.com/codeflow?start=8&length=25)

Units?

Reply:
this depends on the quota bucket on which free trial is applied. Added eg. for now.